db/migrate/20221004161954_add_users_to_target.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddUsersToTarget < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :targets, :user, index: true
```

Comment:
What about adding this to the create_targets migrations?
We should not change migrations when they are merged, but since this is your branch, I think it's ok to do it

Reply:
Sure, I will include this in the target migration!